### PR TITLE
Feature/seed data migrations

### PR DESCRIPTION
**Pull request by ajolson7**

### CODE REVIEW

File: migrations/20180728211117-addBaseDataColumns.js [hunk 1]
```diff
@@ -203,10 +228,15 @@ exports.down = function(db, callback) {
       db.removeColumn.bind(db, 'company_addresses', 'zipcode'),
       // properties table
       db.removeColumn.bind(db, 'properties', 'name_property'),
+      // property addresses
+      db.removeColumn.bind(db, 'property_addresses', 'address_line_1'),
+        db.removeColumn.bind(db, 'property_addresses', 'address_line_2'),
```
Comment (adamsonb12) [inline migrations/20180728211117-addBaseDataColumns.js:233]: Is this indented here by a couple extra spaces?

File: migrations/20180804184736-testSeedData.js [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+exports.up = function(db, callback) {
+  async.series(
+    [
+      // companies
+      db.insert.bind(db, 'companies',
+        ['id', 'name_company', 'shortname_company'],
+        [company_id1, 'Martha & Company', 'Marta & Co.']
+      ),
+        db.insert.bind(db, 'companies',
```
Comment (adamsonb12) [inline migrations/20180804184736-testSeedData.js:99]: I think these are indented again for some reason.

File: migrations/20180804184736-testSeedData.js [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+    callback
+  );
+};
+
+exports.down = function(db, callback) {
+  async.series(
+    [
+      db.runSql.bind(db, `DELETE FROM companies`),
+      db.runSql.bind(db, `DELETE FROM company_addresses`),
+        db.runSql.bind(db, `DELETE FROM crews`),
```
Comment (adamsonb12) [inline migrations/20180804184736-testSeedData.js:337]: Odd indentation again

**Review by adamsonb12 — APPROVED** [review]
The code looks pretty great. Approved. My comments are all on odd indentation which can be fixed by another quick push, or when I eventually set up an auto formatter for our repo someday.

File: migrations/20180728211117-addBaseDataColumns.js (hunk 1 above)
Comment (ajolson7) [reply migrations/20180728211117-addBaseDataColumns.js:233]: Yeah, my IDE was giving me weird spacing issues, I thought I had those taken care of, my bad.

**Pull request merged**